src/main/java/com/cgdp/library/service/AuthorService.java

```diff
@@ -0,0 +1,24 @@
+package com.cgdp.library.service;
+
+import com.cgdp.library.entity.AuthorEntity;
+import com.cgdp.library.repository.AuthorRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@AllArgsConstructor
+public class AuthorService {
+
+    private AuthorRepository authorRepository;
+
+    @Transactional
+    public AuthorEntity save(String authorName) {
```

Comment:
Rename to `createAuthor` to be more accurate, return `AuthorDTO`

```suggestion
    public AuthorDTO createAuthor(String authorName) {
```